### ISSUE TITLE
Cannot open new window after closing all windows

### ISSUE THREAD

**Issue by Khobalt** [microsoft/vscode#100406]
- VSCode Version: 
1.46.0
a5d1cc28bb5da32ec67e86cc50f84c67cc690321
x64

- OS Version: 
OSX Catalina 10.15.5

![image](https://user-images.githubusercontent.com/848683/84923240-d1782780-b07b-11ea-9bd0-d55a883849de.png)

Steps to Reproduce:

1. Open VS Code
2. CMD-SHIFT-W
3. CMD-SHIFT-N

Does this issue occur when all extensions are disabled?: Yes/No
Yes

Notes:
This was working until I factory reset my Mac and then reinstalled VS Code. I've done a (thorough) reinstallation of VS Code, and the bug persists.

Tail of verbose log:
```
[main 2020-06-17T15:43:27.907Z] Lifecycle#window.on('close') - window ID 1
[main 2020-06-17T15:43:27.907Z] Lifecycle#unload() - window ID 1
[1738:0617/084328.010406:INFO:CONSOLE(1669)] "%cTRACE color: #888 lifecycle: onBeforeUnload (reason: 1)", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.010472:INFO:CONSOLE(1669)] "%cTRACE color: #888 lifecycle: onBeforeUnload continues without veto", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[main 2020-06-17T15:43:28.010Z] Lifecycle#unload() - no veto (window ID 1)
[1738:0617/084328.012259:INFO:CONSOLE(1669)] "%cTRACE color: #888 lifecycle: onWillUnload (reason: 1)", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.017431:INFO:CONSOLE(1669)] "%cTRACE color: #888 [storage state.vscdb] updateItems(): insert(Map(1) {lifecyle.lastShutdownReason => 1}), delete(Set(0) {})", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.017854:INFO:CONSOLE(1669)] "%cTRACE color: #888 [storage state.vscdb] Trace (event): BEGIN TRANSACTION", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.018040:INFO:CONSOLE(1669)] "%cTRACE color: #888 [storage state.vscdb] Trace (event): INSERT INTO ItemTable VALUES ('lifecyle.lastShutdownReason','1')", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.018568:INFO:CONSOLE(1669)] "%cTRACE color: #888 [storage state.vscdb] Trace (event): END TRANSACTION", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[1738:0617/084328.024945:INFO:CONSOLE(1669)] "%cTRACE color: #888 [storage state.vscdb] close()", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js (1669)
[main 2020-06-17T15:43:28.033Z] Lifecycle#onBeforeWindowClose.fire() - window ID 1
[main 2020-06-17T15:43:28.034Z] Lifecycle#window.on('closed') - window ID 1
[main 2020-06-17T15:43:30.291Z] [uncaught exception in main]: TypeError: Cannot read property 'id' of undefined
[main 2020-06-17T15:43:30.291Z] TypeError: Cannot read property 'id' of undefined
    at fallbackMenuHandlers.workbench.action.newWindow (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:542:489)
    at t.click (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:555:626)
    at MenuItem.click (electron/js2c/browser_init.js:1572:9)
    at Function.executeCommand (electron/js2c/browser_init.js:1833:13)
[main 2020-06-17T15:43:30.292Z] telemetry/workbenchActionExecuted { id: 'workbench.action.newWindow', from: 'menu' }
[main 2020-06-17T15:43:45.345Z] update#checkForUpdates, state =  idle
[main 2020-06-17T15:43:45.345Z] update#setState checking for updates
[main 2020-06-17T15:43:45.460Z] update#setState idle
[main 2020-06-17T15:43:45.461Z] telemetry/update:notAvailable { explicit: false }
[main 2020-06-17T16:15:01.480Z] [uncaught exception in main]: TypeError: Cannot read property 'id' of undefined
[main 2020-06-17T16:15:01.480Z] TypeError: Cannot read property 'id' of undefined
    at fallbackMenuHandlers.workbench.action.newWindow (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:542:489)
    at t.click (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:555:626)
    at MenuItem.click (electron/js2c/browser_init.js:1572:9)
    at Function.executeCommand (electron/js2c/browser_init.js:1833:13)
[main 2020-06-17T16:15:01.481Z] telemetry/workbenchActionExecuted { id: 'workbench.action.newWindow', from: 'menu' }
**[main 2020-06-17T16:15:56.476Z] [uncaught exception in main]: TypeError: Cannot read property 'id' of undefined
[main 2020-06-17T16:15:56.476Z] TypeError: Cannot read property 'id' of undefined
    at fallbackMenuHandlers.workbench.action.newWindow (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:542:489)
    at t.click (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:555:626)
    at MenuItem.click (electron/js2c/browser_init.js:1572:9)
    at Function.executeCommand (electron/js2c/browser_init.js:1833:13)
[main 2020-06-17T16:15:56.477Z] telemetry/workbenchActionExecuted { id: 'workbench.action.newWindow', from: 'menu' }**
```

**Comment by gjsjohnmurray** [comment]
/duplicate of #99998 and will be fixed in upcoming 1.46.1

**Comment by Khobalt** [comment]
Ah got it. Yeah, I just built VS Code from source and it's working there.
